src/provider/view/NotificationCenterApp.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {Header} from './components/Header/Header';
+import {NotificationView} from './components/NotificationView/NotificationView';
+import {Footer} from './components/Footer/Footer';
+import {setup} from './setup';
+import {NotificationCenter} from '../controller/NotificationCenter';
+
+interface AppState {
+    groupBy: GroupByType;
+    handleGroupBy: (groupBy: GroupByType) => void;
+}
+
+export enum GroupByType {
```

Comment:
I know that this is inline with the pattern we use elsewhere for enums of type properties (i.e. `[varName]Type), but this one parses weirdly to me. I originally read it as Group-ByType instead of GroupBy-Type. Not got a good idea currently what it should be, but can we change some variable/type names here or there to make it more clear?

Reply:
Perhaps `GroupingMethod` or `GroupByMethod`?

Reply:
Split the difference and do `GroupingType`?

Reply:
Sure.